modules/rosa-policy-incident.adoc

```diff
@@ -102,20 +102,22 @@ Platform audit logs are securely forwarded to a centralized security information
 
 [id="rosa-policy-incident-management_{context}"]
 == Incident management
-An incident is an event that results in a degradation or outage of one or more Red{nbsp}Hat services. An incident can be raised by a customer or a Customer Experience and Engagement (CEE) member through a support case, directly by the centralized monitoring and alerting system, or directly by a member of the SRE team.
+An incident is an event that results in a degradation or outage of one or more Red{nbsp}Hat services, and can affect Service Level Agreements (SLAs). An incident can be raised by a customer or a Customer Experience and Engagement (CEE) member through a support case, directly by the centralized monitoring and alerting system, or directly by a member of the SRE team.
```

Comment:
This suggestion changes SLA to standard grammar, but I'm not sure if we use something different at RH as an official term though

```suggestion
An incident is an event that results in a degradation or outage of one or more Red{nbsp}Hat services, and can affect service-level agreements (SLAs). Customers and Customer Experience and Engagement (CEE) members can raise an incident through a support case. The centralized monitoring and alerting system and members of the SRE team can raise an incident directly.
```

Note: "directly" is not really clear to me here. I am guessing it means "without a support case", but I don't know if there's a better way to say what it actually means in practice?